app/javascript/flavours/glitch/features/account_timeline/components/header.jsx

```diff
@@ -153,10 +158,16 @@ class Header extends ImmutablePureComponent {
           <div className='account__section-headline'>
             <NavLink exact to={`/@${account.get('acct')}`}><FormattedMessage id='account.posts' defaultMessage='Posts' /></NavLink>
             <NavLink exact to={`/@${account.get('acct')}/with_replies`}><FormattedMessage id='account.posts_with_replies' defaultMessage='Posts and replies' /></NavLink>
-            <NavLink exact to={`/@${account.get('acct')}/featured`}><FormattedMessage id='explore.trending_tags' defaultMessage='Hashtags' /></NavLink>
+            <NavLink to={`/@${account.get('acct')}/featured`}><FormattedMessage id='account.featured' defaultMessage='Featured' /></NavLink>
             <NavLink exact to={`/@${account.get('acct')}/media`}><FormattedMessage id='account.media' defaultMessage='Media' /></NavLink>
           </div>
         )}
+        {!(hideTabs || hidden) && featured && (
+          <div className='account__section-headline'>
```

Comment:
I might want to add an extra class here to allow custom themes to style this differently.